frontend/spec/controllers/spree/checkout_controller_spec.rb

```diff
@@ -170,11 +170,6 @@ def spree_post_address
           expect(response).to redirect_to spree.order_path(order)
         end
 
-        it 'populates the flash message' do
-          post :update, params: { state: 'confirm' }
-          expect(flash.notice).to eq(Spree.t(:order_processed_successfully))
```

Comment:
This flash message is not being set in the controller anymore

Reply:
@KirillKayumov setting this flash message is essential for conversion tracking - https://github.com/spree/spree/blob/230f6d8e780c5b4ba4173956e83026650e01b4db/frontend/app/helpers/spree/orders_helper.rb#L4

We cannot remove it, we need to preserve it but not display it on Order Confirmation

see https://github.com/spree-contrib/spree_analytics_trackers/search?q=order_just_completed%3F%28%40order%29&unscoped_q=order_just_completed%3F%28%40order%29

Reply:
@damianlegawiec thanks but we still have this flag in `flash` https://github.com/spark-solutions/spree/blob/f3fa6bd35364754d9deb765400db823ea980d578/frontend/app/controllers/spree/checkout_controller.rb#L42
so it should work well